README.md

```diff
@@ -386,6 +386,34 @@ $ helm install my-release \
     neotys/nlweb
 ```
 
+#### Side-car containers (experimental)
```

Comment:
As we discussed, my main feeling remains that this doc should be titled by what it actually does, something like : 
"How to handle traffic with custom nginx reverse proxy using side-cars"
or even better
"How to implement SSL termination at the pod level using nginx side-cars"

Reply:
I renamed it _How to handle traffic with custom nginx reverse proxy using side-cars_
Main README was already correct.

Reply:
My bad, I was thinking we were talking about the example README.

Here, I would prefer to keep more or less like this, because overall the feature is about adding side-car containers to the NLW deployment.
The provided example is about adding a nginx proxy to demonstrate how this feature can be used.